pydicom/config.py

```diff
@@ -385,3 +390,42 @@ def debug(debug_on=True, default_handler=True):
 
 # force level=WARNING, in case logging default is set differently (issue 103)
 debug(False, False)
+
+_use_future = False
```

Comment:
Why make this "private" if you want to access it via `config`?

Reply:
I was interpreting "private" as meaning 'internal to pydicom'.  User code should only use `future_behavior()`, in case of future changes to how this works.

Reply:
Ok. I'm just not to do this as pylint does not like it - but I see your point.